### PR TITLE
AV-203851 documentation for controlPlaneHmOnly flag

### PR DESCRIPTION
**Pull request by arihantg**

### CODE REVIEW

File: docs/crds/gdp.md [hunk 1]
```diff
@@ -118,12 +121,14 @@ matchRules:
 
 11. `pkiProfileRef`: Provide an PKI Profile ref (pre-created in Avi Controller).This has to be a federated profile. It will be applied only if sitePersistence is enabled.
 
+12. `controlPlaneHmOnly`: If this boolean flag is set to `true` Only control plane health monitoring will be done. Amko will not add any `healthMonitorRefs` or create any data plane health monitors. It is `false` by default.
```
Comment (akshayhavile) [inline docs/crds/gdp.md:124]: `Amko` --> `AMKO`

Comment (akshayhavile) [inline docs/crds/gdp.md:124]: Please change to `If ... set to 'true', only control ...`

Comment (arihantg) [reply docs/crds/gdp.md]: updated

Comment (arihantg) [reply docs/crds/gdp.md]: updated

**Review by akshayhavile — APPROVED** [review]
LGTM

**Pull request merged**